src/panel/pages/explorer/components/Icons.tsx

```diff
@@ -35,20 +29,10 @@ export const SeeMoreIcon = ({ className }: { className?: string }) => (
   </svg>
 );
 
-export const CacheOutcomeIcon = ({
-  className,
-  state
-}: {
-  className?: string;
-  state: "hit" | "miss" | "partial" | undefined;
-}) => (
-  <svg
-    width="10"
-    height="10"
-    viewBox="0 0 10 10"
-    fill="none"
-    className={className}
-  >
+export const CacheOutcomeIcon: FC<JSX.IntrinsicElements["svg"] & {
```

Comment:
Type wizard

Reply:
I did this because we could then animate the icon rotating

Reply:
Didn't get around to animating yet!